azure-communication-ui/calling/src/main/java/com/azure/android/communication/ui/calling/presentation/fragment/setup/SetupViewModel.kt

```diff
@@ -18,6 +19,7 @@ internal class SetupViewModel(
     store: Store<ReduxState>,
     setupViewModelProvider: SetupViewModelFactory,
     private val networkManager: NetworkManager,
+    private val callType: CallType? = null,
```

Comment:
Can have default value, or mandatory? Ways to non-null?

Reply:
As per discussion we will fix for GA.